base/doc/ltnews35.tex

```diff
@@ -323,6 +323,22 @@ \subsection{???}
 
 \section{Code improvements}
 
+\subsection{\cs{protected} UTF-8 character definitions}
+The characters defined via \file{utf8.def} are now defined as \cs{protected}
+macros. This makes them safe to use in expansion contexts where the
+classic \cs{protect} mechanism is not enabled, notably L3 programming
+layer \texttt{e} and \texttt{x} arguments.
+
```

Comment:
and file name parsing

Reply:
actually filename parsing is already protcted via ifcsname but edef, e  and x have nothing without \protected